rocAL/source/pipeline/master_graph.cpp

```diff
@@ -1056,9 +1338,19 @@ std::vector<rocalTensorList *> MasterGraph::create_coco_meta_data_reader(const c
         auto bbox_info = default_bbox_info;
         _labels_tensor_list.push_back(new Tensor(labels_info));
         _bbox_tensor_list.push_back(new Tensor(bbox_info));
-        if (metadata_type == MetaDataType::PolygonMask) {
+        if (metadata_type == MetaDataType::PolygonMask || metadata_type == MetaDataType::PixelwiseMask) {
             auto mask_info = default_mask_info;
             _mask_tensor_list.push_back(new Tensor(mask_info));
+            if (metadata_type == MetaDataType::PolygonMask) {
+                auto select_mask_polygon_info = default_select_mask_polygon_info;
+                _select_mask_polygon_list.push_back(new Tensor(select_mask_polygon_info));
+            }
+            if (metadata_type == MetaDataType::PixelwiseMask) {
+                auto random_mask_pixel_info = default_random_mask_pixel_info;
+                _random_mask_pixel_list.push_back(new Tensor(random_mask_pixel_info));
+                auto random_object_bbox_info = default_random_object_bbox_info;
+                _random_object_bbox_list.push_back(new Tensor(random_object_bbox_info));
```

Comment:
Please ensure to release the memory allocated for this new tensor in the release function of master graph

https://github.com/SundarRajan98/rocAL/blob/9af23be0fc470c4b2b7a207f90526ccb0e12fc07/rocAL/source/pipeline/master_graph.cpp#L348

I could see vx tensor is getting release here, not sure if the actual tensor is getting freed also